pyzeebe/worker/task_router.py

```diff
@@ -1,24 +1,25 @@
 import logging
-from typing import Callable, List, Tuple, Optional
+from typing import Callable, List, Optional, Tuple
```

Comment:
If it's desired that people follow the rules around import that's followed with this commit, I think adding `isort` in pre-commit and GH Actions is needed.